Userland/Libraries/LibJS/Tests/builtins/String/String.prototype.localeCompare.js

```diff
@@ -9,7 +9,7 @@ test("basic functionality", () => {
         const aTob = a.localeCompare(b);
         const bToa = b.localeCompare(a);
 
-        expect(aTob).toBe(1);
+        expect(aTob > 0);
```

Comment:
This doesn't actually do anything, need to call `toBetrue()`. Also, why can't we check the actual value?

Same below.

Reply:
This function (I copied it from `localeCompare`) receives arbitrary input, so we can't know the actual value, fixed the expect issue